chapters/data-collection.tex

```diff
@@ -1,362 +1,251 @@
+The Data-focused pilot should be done in advance of Enumerator training
 
-\section{Overseeing fieldwork and quality assurance}
-
-While the team is in the field, the research assistant
-and field coordinator will be jointly responsible
-for making sure that the survey is progressing correctly,\sidenote{\url{https://dimewiki.worldbank.org/wiki/Data_Quality_Assurance_Plan}}
-that the collected data matches the survey sample,
-and that errors and duplicate observations are resolved
-quickly so that the field team can make corrections.\sidenote{\url{https://dimewiki.worldbank.org/wiki/Duplicates_and_Survey_Logs}}
-Modern survey software makes it relatively easy
-to control for issues in individual surveys,
-using a combination of in-built features
-such as hard constraints on answer ranges
-and soft confirmations or validation questions.
-
-These features allow you to spend more time
-looking for issues that the software cannot check automatically.\sidenote{\url{https://dimewiki.worldbank.org/wiki/Monitoring_Data_Quality}}
-Namely, these will be suspicious patterns across multiple responses
-or across a group of surveys rather than errors in any single response field
-(those can often be flagged by the questionnaire software);
-enumerators who are taking either too long or not long enough to complete their work,
-``difficult'' groups of respondents who are systematically incomplete; and
-systematic response errors.
-These are typically done in two main forms:
-high-frequency checks (HFCs) and back-checks.
-
-\textbf{High-frequency checks} are carried out on the data side.\sidenote{\url{https://github.com/PovertyAction/high-frequency-checks/wiki}}
-First, observations need to be checked for duplicate entries:
+
+
+%------------------------------------------------
+\section{Data quality assurance}
+A huge advantage of CAPI surveys, compared to traditional paper surveys, is the ability to access and analyze the data while the survey is ongoing.
+Data issues can be identified and resolved in real-time. Designing systematic data checks, and running them routinely throughout data collection, simplifies monitoring and improves data quality.
+As part of survey preparation, the research team should develop a \textbf{data quality assurance plan} \sidenote{\url{https://dimewiki.worldbank.org/wiki/Data_Quality_Assurance_Plan}}.
+While data collection is ongoing, a research assistant or data analyst should work closely with the field team to ensure that the survey is progressing correctly, and perform \textbf{high-frequency checks (HFCs)} of the incoming data.
+\sidenote{\url{https://github.com/PovertyAction/high-frequency-checks/wiki}}
+Data quality assurance requires a combination of both real-time data checks, survey audits, and field monitoring. Although field monitoring is critical for a successful survey, we focus on the first two in this chapter, as they are the most directly data related.
```

Comment:
No problem that we do not focus on _field monitoring_ but if we mention it I think we need to say something about it.